.github/workflows/x-env.yml

```diff
@@ -0,0 +1,34 @@
+name: X Environment setup
+
+on:
+  workflow_call:
+    outputs:
+      gh-org:
+        description: "GitHub Organization"
+        value: ${{ jobs.env.outputs.gh-org }}
+      mvn-releases-url:
+        description: "Maven Releases URL"
+        value: ${{ jobs.env.outputs.mvn-releases-url }}
+      mvn-snapshots-url:
+        description: "Maven Snapshots URL"
+        value: ${{ jobs.env.outputs.mvn-snapshots-url }}
+      deploy-skip:
+        value: ${{ jobs.env.outputs.depoly-skip }}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  env:
+    runs-on: ubuntu-latest
+    outputs:
+      gh-org: ${{ env.gh-org }}
+      mvn-releases-url: ${{ env.mvn-releases-url }}
+      mvn-snapshots-url: ${{ env.mvn-snapshots-url }}
+      deploy-skip: ${{ env.deploy-skip }}
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Same here, the main workflow should be responsible for checking out the code, not a re-usable action.

Reply:
Fixed in follow-up commit.